pkgs/development/python-modules/ffcv/default.nix

```diff
@@ -47,16 +56,58 @@ buildPythonPackage rec {
     tqdm
   ];
 
-  # `ffcv._libffcv*.so` cannot be loaded in the nix build environment for some
-  # reason. See https://github.com/NixOS/nixpkgs/pull/160441#issuecomment-1045204722.
-  doCheck = false;
-
   pythonImportsCheck = [ "ffcv" ];
 
-  meta = with lib; {
+  # C/C++ python modules are only in the installed output and not in the build
+  # directory. Since tests are run from the build directory python prefers to
+  # import the local module first which does not contain the C/C++ python
+  # modules and results in an import error. By changing the directory to
+  # 'tests' the build directory is no long available and python will import
+  # from the installed output in the nix store which does contain the C/C++
+  # python modules.
+  preCheck = ''
+    cd tests
+  '';
+
```

Comment:
```suggestion
  postCheck = ''
    cd ..
  '';
  
```

Reply:
I think it's better to leave it in the tests directory. We don't have to append any extra paths to the disabled tests and pytest will automatically find all the tests.

Reply:
> I think it's better to leave it in the tests directory. We don't have to append any extra paths to the disabled tests and pytest will automatically find all the tests.

There are are phases and hooks executed after `checkPhase`; there's no "good" reason they shouldn't access the build directroy again, but they just might; a user of `.overrideAttrs` might find this surprising